app/views/external_users/certifications/new.html.haml

```diff
@@ -110,5 +110,5 @@
         = f.gov_uk_date_field(:certification_date, form_hint_text: '', legend_text: t('.date'), legend_class: 'govuk-legend', id: 'certification_date')
 
     .button-holder
-      = f.submit t('.submit'), class: 'button left btn-spacer-20'
-      = link_to t('.return'), edit_polymorphic_path(@claim), class: 'button-gray-3 left'
+      = govuk_button() { t('.submit') }
```

Comment:
[ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3605/files#r526000031)